### PR TITLE
Added solved problem in NN&CV_zadanie

### PR DESCRIPTION
**Pull request by Tczek**
Added solved problem in NN&CV_zadanie

Feedback highly anticipated.

### CODE REVIEW

**Review by Maxoo005 — COMMENTED** [review]
Hejj, bardzo przejrzyste i logiczne wykonanie, sensowne wizualizacje wyników. Bardzo solidnie. Dużo wyniosłem przeglądając twoją prace.

**Comment by anbonik** [comment]
Bardzo dobre rozwiązanie. Fajnie , że opisałeś jak działa CNN , to udało mi się dzięki tobie lepiej to zrozumieć. Ciekawe są wnioski z rozpoznawania poszczególnych liczb. Mógłbyś jeszcze dodać testy z sieci o różnych wymiarach , dropout i batchnorm.

**Review by nuklearbanana — APPROVED** [review]
Bardzo szczegółowe i czytelne opisy na +. 
Można było jeszcze eksperymentować z różnymi kombinacjami warstw, parametrów i epok. 
Jeszcze raz + za czytelność i opisanie kodu.

**Review by lolakk05 — APPROVED** [review]
Dobrze wykonane zadanie, wszystko naprawdę bardzo porządnie opisane i fajnie zaznaczone nwm liniami. Kod też czytelny więc nie ma się do czego przyczepić ;)

**Comment by Karo555** [comment]
Hej, 
poniżej sugestie i analiza.
1) brakuje pre-processingu danych
2) błędy w nazewnictwie: 
`self.fcl = nn.Linear(320, 50)`
`self.fc2 = nn.Linear(50, 10)`
raz mamy **fcl**, a raz **fc2**. lepiej nazywać fc1 i fc2 lub fcl1 i fcl2
